src/set_relation/UFCallMap.cc

```diff
@@ -64,50 +64,60 @@ string UFCallMap::symUFC( std::string &ufcName )
 
 //! Inserts a UFC term to both of themaps.
 //  The function creates an string representing the UFC as symbolic constant,
-//  then,  adds (ufc,str) to mUFC2Str & adds (ufc,str) to mStr2UFC
-//  It does not add repetitive UFCs
-void UFCallMap::insert( UFCallTerm *ufcterm )
+//  then, adds (ufc,vt) & (vt,ufc) to mUFC2VarParam & mVarParam2UFC
+//  It does not add repetitive UFCs. The function does not own ufcterm.
+void UFCallMap::insert( UFCallTerm *ufc )
```

Comment:
Make sure to have the comments for these methods indicate that the coefficient is being ignored.